schema/data/portal/reporting_year.sql

```diff
@@ -0,0 +1,14 @@
+begin;
+
+with rows as (
+insert into ggircs_portal.reporting_year(reporting_year, reporting_period_start, reporting_period_end, application_open_date, application_end_date, application_response_date)
+overriding system value
+values
+(2018, '2017-01-01 14:49:54.191757-07', '2018-12-30 14:49:54.191757-07', '2018-04-01 14:49:54.191757-07', '2018-12-30 14:49:54.191757-07', '2019-01-30 14:49:54.191757-07'),
+(2019, '2018-01-01 14:49:54.191757-07', '2019-12-30 14:49:54.191757-07', '2019-04-01 14:49:54.191757-07', '2019-12-30 14:49:54.191757-07', '2020-01-30 14:49:54.191757-07')
+on conflict(reporting_year) do update set
+reporting_year=excluded.reporting_year
```

Comment:
in the list of columns to update on conflict, you need to have everything **but** the column that is conflicting.

What you wrote is telling the database: "if there is already a row with the `reporting_year` 2018, set the reporting year to 2018". 
What you want is "if there is already a row with the `reporting_year` 2018, set `reporting_period_start` to the value we tried to insert (`excluded.reporting_period_start`), etc for every column other than the primary key.